### PR TITLE
[luci-interpreter] Add RmsNorm to loader

### PR DESCRIPTION
**Pull request by seockho-kim**
This commit adds RmsNorm to luci-interpreter loader

ONE-DCO-1.0-Signed-off-by: Seockho Kim seockho.kim@samsung.com

issue: https://github.com/Samsung/ONE/issues/13964
draft: https://github.com/Samsung/ONE/pull/13967

### CODE REVIEW

File: compiler/luci-interpreter/pal/linux/KernelsToBuild.lst [hunk 1]
```diff
@@ -66,6 +66,7 @@ REGISTER_KERNEL(Reshape)
 REGISTER_KERNEL(ResizeBilinear)
 REGISTER_KERNEL(ResizeNearestNeighbor)
 REGISTER_KERNEL(ReverseV2)
+REGISTER_KERNEL(RmsNorm)
```
Comment (seockho-kim) [inline compiler/luci-interpreter/pal/linux/KernelsToBuild.lst:69]: It was submitted as another PR.
But, it's added here because of link error.

**Review by jinevening — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM
